Engine/DataFeeds/ZipDataCacheProvider.cs

```diff
@@ -266,7 +271,12 @@ public class CachedZipFile : IDisposable
         /// <summary>
         /// The ZipFile this object represents
         /// </summary>
-        public ZipFile ZipFile { get; }
+        private readonly ZipFile _zipFile;
+
+        /// <summary>
+        /// Contains all entries of the zip file by filename
+        /// </summary>
+        public Dictionary<string, ZipEntry> EntryCache = new Dictionary<string, ZipEntry>(StringComparer.OrdinalIgnoreCase);
```

Comment:
nit - should probably be marked as `readonly` -- maybe even `IReadOnlyDictionary`

Reply:
done 👍 . For some reason ReSharper didn't notice, I guess I can't blindly trust it for such things. Using `IReadOnlyDictionary` didn't allow me to add items in the constructer

Reply:
could have been done like this:
```
public CachedZipFile( ... )
{
    var entryCache = new Dictionary<string, ZipEntry>();
    // add entries
    EntryCache = entryCache;;
}
```